.github/workflows/ci.yml

```diff
@@ -28,8 +28,8 @@ jobs:
       - uses: actions/checkout@v3
 
       - name: Remove Gemfile.lock
-        if: matrix.os == 'windows-latest' && matrix.ruby == '3.1'
-        run: rm Gemfile.lock
```

Comment:
This was invalid previously because it was removing the wrong lockfile in cases where the gemfile was `Gemfile-rails-main`. It seems the `gemspec` entry in the Gemfile wasn't playing well with a Gemfile contained in a subdirectory, so I moved it up to the root.